### PR TITLE
chore(editor-package): Pull katex-styles from npm serlo org

### PR DESCRIPTION
**Pull request by CodingDive**

### CODE REVIEW

File: packages/editor/package.json [hunk 1]
```diff
@@ -39,9 +39,9 @@
     "yalc:publish": "yarn build && yalc publish --push --sig"
   },
   "dependencies": {
+    "@serlo/katex-styles": "^1.0.0",
```
Comment (elbotho) [inline packages/editor/package.json]: why is this package in [a different repo](https://github.com/serlo/serlo-katex-styles) and not part of the monorepo?

Comment (elbotho) [reply packages/editor/package.json]: (then we could also use it with `workspace:*` here)

Comment (CodingDive) [reply packages/editor/package.json]: > (then we could also use it with workspace:* here)

Not with this package, but I ran into some trouble using a yarn workspace as a normal (non dev) dependency and then publishing to npm. It broke npm/pnpm in resolving the dependency tree when I was trying to install from the registry.

That's not to say, I can't make this part of the monorepo but we would probably have to use vite & ts paths to alias the import from the monorepo package, have another package.json + all the config boiler plate files and run the risk of having more of a Eslint/TypeScript mess.

I'd prefer it in an extra repo if it's not a file we are touching too often.

Comment (elbotho) [reply packages/editor/package.json]: Hmm I don't veto this. But I want to make sure…
In my opinion small packages like that are a perfect fit for the monorepo concept and part of the reason why we set it up.
We don't have to use `workspace:*` as the source if that makes it easier.

Comment (CodingDive) [reply packages/editor/package.json]: Cool! Will try to set it up tomorrow :blush:

**Review by elbotho — APPROVED** [review]
thanks for moving it into the monorepo :)

**Pull request merged**